webapp/src/features/UserItem/ActivityHistory.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react'
+import { Row, Col, Button, Image } from 'react-bootstrap'
+import Icons from '../../assets/icons'
+import Table from '../../widgets/Table'
+
+const ActivityHistory = () => {
+	const creditIconFormatter = (row) => {
```

Comment:
Looks like we haven't use SVG Icons directly and may be we can re-use alot of code there,
Will discuss
@aakashThapa0011

Reply:
updated